### PR TITLE
Bump minio version

### PR DESCRIPTION
**Pull request by PennyScissors**
Bump minio version in image-list

Related Issue: rancher/rancher#27467

### CODE REVIEW

**Comment by maggieliu** [comment]
related to: https://github.com/rancher/rancher/issues/27467

File: images-list [hunk 1]
```diff
@@ -15,7 +15,7 @@ quay.io/coreos/prometheus-operator rancher/coreos-prometheus-operator v0.38.1
 quay.io/coreos/prometheus-config-reloader rancher/coreos-prometheus-config-reloader v0.38.1
 quay.io/coreos/kube-state-metrics rancher/coreos-kube-state-metrics v1.9.5
 plugins/docker rancher/plugins-docker 18.09
-minio/minio rancher/minio-minio RELEASE.2020-06-03T22-13-49Z
+minio/minio rancher/minio-minio RELEASE.2020-07-02T00-15-09Z
```
Comment (prachidamle) [inline images-list:18]: I see few more recent tags/releases under https://github.com/minio/minio/tags

Don't we want to update to the latest?

**Comment by deniseschannon** [comment]
@prachidamle Good catch. yes, we should update to the latest @PennyScissors

**Comment by PennyScissors** [comment]
@deniseschannon I updated it to the current latest.

**Pull request merged**